x-pack/plugins/security_solution/public/siem_migrations/rules/pages/index.tsx

```diff
@@ -34,13 +34,11 @@ export const RulesPage: React.FC<RulesMigrationPageProps> = React.memo(
 
     const { data: ruleMigrationsStatsAll, isLoading: isLoadingMigrationsStats } = useLatestStats();
 
-    const migrationsIds = useMemo(() => {
+    const filnishedRuleMigrationsStats = useMemo(() => {
```

Comment:
Typo
```suggestion
    const finishedRuleMigrationsStats = useMemo(() => {
```
I use [this extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to prevent that

Reply:
ooppsss 😄 thanks for the extension link!!